subsys/bluetooth/host/id.c

```diff
@@ -401,7 +403,12 @@ static void adv_disable_rpa(struct bt_le_ext_adv *adv, void *data)
 	/* Disable advertising sets to prepare them for RPA update. */
 	if (atomic_test_bit(adv->flags, BT_ADV_ENABLED) &&
 	    !atomic_test_bit(adv->flags, BT_ADV_USE_IDENTITY)) {
-		bt_le_adv_set_enable_ext(adv, false, NULL);
+		int err;
+
+		err = bt_le_adv_set_enable_ext(adv, false, NULL);
+		if (err) {
+			BT_ERR("Failed to disable advertising\n");
```

Comment:
Would be good to include the value of err in the log (especially since the function doesn’t pass it back to the caller)